grommet/hpe-design-system
# Remove redundant Error messages

#### What does this PR do?

Removing redundant uses of custom `Error` component that are now being handled by the theme. I have also removed the local theme changes now that they have merged into grommet-theme-hpe.

#### Where should the reviewer start?
Any file.

#### What testing has been done on this PR?
Tested locally.

#### How should this be manually tested?
Check deploy on changed files. Only one `CircleAlert` icon should appear on the error state.

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, backwards compatible.
